### DISCUSSION TITLE
Missing Cursor in Nvim Tree

### DISCUSSION THREAD

**Discussion by chikko80** [olimorris/onedarkpro.nvim#176] [Q&A] [answered]
Hi, 
I am not sure if this feature I missing or it's just a configuration thing but if I switch to this color scheme:

[navarasu/onedark.nvim](https://github.com/navarasu/onedark.nvim)

i get a nice little cursor in my nvim-tree that follows my current file (since i've activated sync_root_with_cwd = true).

i completely miss the "highlighted cursor/line" on the current open file in the onedarkpro scheme.

This is how it looks:
<img width="984" alt="Screenshot 2023-03-26 at 2 51 18 PM" src="https://user-images.githubusercontent.com/58805871/227778928-d591609a-618a-42aa-93d5-c99645a72cb3.png">

But i can't reproduce it with the onedarkpro.nvim - i've even tried the minimal lua:

```lua

local test_dir = "/tmp/onedarkpro"
local root = vim.fn.fnamemodify(test_dir, ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "--single-branch",
    "https://github.com/folke/lazy.nvim.git",
    lazypath,
  })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "olimorris/onedarkpro.nvim",
  { "nvim-treesitter/nvim-treesitter", build = ":TSUpdate" },

  'nvim-tree/nvim-tree.lua',

  -- add any other plugins here
}

require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

-- setup treesitter
local ok, treesitter = pcall(require, "nvim-treesitter.configs")
if ok then
  treesitter.setup({
    ensure_installed = "all",
    ignore_install = { "phpdoc" }, -- list of parser which cause issues or crashes
    highlight = { enable = true },
  })
end

-- setup onedarkpro
local ok, onedarkpro = pcall(require, "onedarkpro")
if ok then
  onedarkpro.setup({
    -- Your onedarkpro config here
  })
  vim.cmd("colorscheme onedark")
end


require("onedarkpro").setup({
  highlights = {

  }
})

vim.g.mapleader = " "

vim.cmd("colorscheme onedark")


vim.g.loaded_netrw = 1
vim.g.loaded_netrwPlugin = 1


require("nvim-tree").setup{
    -- keep track of current file
    respect_buf_cwd = true,
    auto_reload_on_write = true,
    sync_root_with_cwd = true,
    update_focused_file = {
      enable = true,
    },
    tab = {
        sync = {
            open = true
        }
    },
    git = {
        enable = true,
        ignore = false,
        timeout = 400,
    },
}

vim.keymap.set("n", "<leader>e", ":NvimTreeToggle<CR>")
```

**Comment by olimorris** [comment]
I'd advise looking at what `onedark.nvim` and this colorscheme both define for `NvimTree` (in terms of highlight groups) and applying it through your own config. Or, posing the question on the `NvimTree` repo.

**Comment by chikko80** [comment, marked as answer]
Got it!
Its related to the setting [CursorLine](https://github.com/olimorris/onedarkpro.nvim/blob/6f9dcb258537d077f6037d3b7a55f6b36f15b1ec/lua/onedarkpro/highlights/editor.lua#L17)

Just override this setting with you favorite background highlight color and you are good to go
